app/src/main/java/kotlinrestfull/yassinelhadedy/github/com/kotlinrx_restfull/MainActivity.kt

```diff
@@ -0,0 +1,43 @@
+package kotlinrestfull.yassinelhadedy.github.com.kotlinrx_restfull
+
+import android.R
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import android.widget.ArrayAdapter
+import android.widget.ListView
+import k2rx.yassinelhadedy.github.com.kotlinretrrx.model.api.RetrofitObject
+import rx.android.schedulers.AndroidSchedulers
+import rx.schedulers.Schedulers
+
+class MainActivity : AppCompatActivity() {
+    var listview: ListView?=null
+    var movies = mutableListOf<String>()
```

Comment:
- Why using mutable list? ummutable list will do.
- movies should be a local variable